main/gradio_ui/builder.py

```
@@ -0,0 +1,208 @@
+            if auto_stop_task and not auto_stop_task.done():
+                auto_stop_task = None
+    except asyncio.CancelledError:
+        return
+
+
+def create_ui():
+    """Create the main Gradio UI for the trading simulation"""
+
+    traders = [
+        Trader(trader_name, lastname, model_name)
+        for trader_name, lastname, model_name in zip(names, lastnames, short_model_names)
```

Ruff (0.13.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)